packages/react-components/react-conformance-griffel/src/overridesWin.ts

```diff
@@ -43,7 +43,7 @@ export const overridesWin: ConformanceTest = (componentInfo, testInfo) => {
   });
 
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
`.resetAllMocks()` also removes implementation while we want only clean stored data.